.github/workflows/code-checks.yml

```diff
@@ -0,0 +1,59 @@
+name: Code Checks
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
This project still uses the `master` branch, can you change these references throughout? We haven't gotten to standardizing on the main branch yet.